### ISSUE TITLE
Report an error for eslint-disable-line comments that span multiple lines

### ISSUE THREAD

**Issue by not-an-aardvark** [eslint/eslint#10334]
**The version of ESLint you are using.**

v4.19.1

**The problem you want to solve.**

It's possible to write a `disable-line` directive comment that spans multiple lines, like this:

```js
/* eslint-disable-next-line
  foo,
  bar,
  baz
*/
```

This is potentially confusing because it's not clear which "line" the directive comment is referring to. In fact, the current behavior is that ESLint ignores the directive comment entirely and doesn't disable anything.

**Your take on the correct solution to problem.**

ESLint should report an error when an `eslint-disable-line` or `eslint-disable-next-line` directive comment spans multiple lines, to inform the user that their directive isn't doing anything so that the user can fix the problem.

This was discussed in https://github.com/eslint/eslint/pull/9745#discussion_r158462665 when block comment support for `disable-line` directives was added. The sentiment at the time was that we couldn't make multi-line `disable-line` directives an error then since it would be a breaking change, so we decided to warn about them at the time with the plan to make them an error in the next major release.

**Comment by not-an-aardvark** [comment]
This issue was accepted in [today's TSC meeting](https://github.com/eslint/tsc-meetings/blob/adef4bed53b4e3160c19e82e7736ee651f540ee0/notes/2018/2018-05-10.md#report-an-error-for-eslint-disable-line-comments-that-span-multiple-lines).
